evap/results/exporters.py

```diff
@@ -72,11 +72,12 @@ def init_grade_styles(cls):
                 "This can happen, if the file is imported / run multiple "
                 "times in one application run.",
                 ImportWarning,
+                stacklevel=3,
```

Comment:
To pinpoint this to the callsite, it should be 2 right? If I duplicate the `ResultsExporter.init_grade_styles()` on line 334, I currently get a warning pointing me to `from evap.results.exporters import TextAnswerExporter`, with 2 it points to my faulty `ResultsExporter.init_grade_styles()`

Reply:
(related: https://github.com/e-valuation/EvaP/pull/2051#discussion_r1433284641 -- we there assumed that an additional execution of this file would be the "root" location, so we wanted to know where it got imported/executed from)

Reply:
Oh okay, sure